README.md

```diff
@@ -3,11 +3,17 @@
   <a href="https://mui.com/" rel="noopener" target="_blank"><img width="150" src="https://mui.com/static/logo.svg" alt="MUI logo"></a>
 </p>
 
-<h1 align="center">MUI</h1>
+<h1 align="center">MUI Core</h1>
 
 <div align="center">
```

Comment:
This creates issues

<img width="1059" alt="Screenshot 2022-04-18 at 23 17 31" src="https://user-images.githubusercontent.com/3165635/163879533-94b15257-c35f-4c1c-b001-a0312d4f48a3.png">